### PR TITLE
Pr 428

### PR DESCRIPTION
**Pull request by cabo**
This is based on main, as github doesn't seem to be able to base on timbray/main, which is the PR this is based on.  Argh.

### CODE REVIEW

**Comment by cabo** [comment]
Please only look at https://github.com/ietf-wg-jsonpath/draft-ietf-jsonpath-base/pull/435/commits/e2c24f1a5a28493b48129fd05f43c0cb8d299685

**Comment by glyn** [comment]
> Please only look at [e2c24f1](https://github.com/ietf-wg-jsonpath/draft-ietf-jsonpath-base/commit/e2c24f1a5a28493b48129fd05f43c0cb8d299685)

Rebased on main after merging Tim's PR. There now remains just the salient commit.

**Comment by cabo** [comment]
OK, resolving with main fixed the "files changed" view.

File: draft-ietf-jsonpath-base.md
Comment (glyn) [inline draft-ietf-jsonpath-base.md:259]: Let's not suggest the a single parameter can take multiple arguments:
```suggestion
: Formal parameter (of a function) that can take a function argument (actual parameter) in a function-expression.
```

File: draft-ietf-jsonpath-base.md
Comment (glyn) [inline draft-ietf-jsonpath-base.md:270]: Hmmm. The above was a vestige of the merge.
```suggestion
```

**Review by glyn — APPROVED** [review]
LGTM, after rebasing and sorting out a small merge issue.

File: draft-ietf-jsonpath-base.md
Comment (cabo) [reply draft-ietf-jsonpath-base.md:259]: Can you copy over this change to #428?
(I can't, because of the weird issue with PRs from "main" on "main".)

**Comment by glyn** [comment]
Looks like you already did that, thanks.

**Pull request merged**
